UITests/Responses/Receipts/ReceiptsForOneMonth.json

```diff
@@ -0,0 +1,91 @@
+{
+    "count": 21,
+    "offset": 0,
+    "limit": 10,
```

Comment:
keep the data consistent. Either use 10 or 20 for limit.

Reply:
fixed